templates/default/rt/rtticketaddbox.html

```diff
@@ -0,0 +1,307 @@
+    <p style="display: none;">
```

Comment:
To by od razu wypadałoby przerobić na div:flexbox.

Reply:
Raczej to się nadaje na oddzielny PR bo tam cała tabelka jest do przerobienia a przerobienie samego nagłówka nic nie da. Pewnie zacznę coś tam dłubać jak tylko sprawdzę dokładnie czy div:flex spełnia moje oczekiwania do końca na interfejsach mobilnych.

Reply:
Nie ma nic lepszego niż flexbox co miałoby już dobre wsparcie we wszystkich przeglądarkach.